time-clock/config/database.yml

```diff
@@ -22,4 +22,6 @@ test:
 
 production:
   <<: *default
-  database: db/production.sqlite3
+  adapter: postgresql
+  database: database_production
+  timeout: 5000
```

Comment:
It is not recommended to leave this file on GitHub, I recommend you place it in the .gitignore, and instead replace it with a database.yml.example file.